### PR TITLE
Improve the logic to fill the sections classes_body and body_data of the page blade template

### PR DESCRIPTION
**Pull request by Shidersz**
The usage of multiple nested ternary operators may lead to confusion and could be hard to maintain. Taking this into account, I have moved the logic that fills the sections **classes_body** and **body_data** of the "page.blade.php" template into **PHP** functions. Those functions return strings values, and then, the sections can filled as:

```
@section('classes_body', getBodyClasses())
@section('body_data', getBodyData())
```

### CODE REVIEW

**Comment by lacodimizer** [comment]
Thanks for your PR! Can we cleanup the file to move the php functions to an smarter solution like a blade view modifier?

**Comment by Shidersz** [comment]
Hi @lacodimizer , I'm not sure if you are asking for permissions to make changes into the code or required extras modification from me. If it is the first case, then yes, you can make changes in the code. If it is the second case, can you give me some insights in what you mean or some references to documentation. I'm very new to Laravel, just learning it while integrating with this package. Thanks!

**Comment by lacodimizer** [comment]
@REJack What do you mean?

File: resources/views/page.blade.php [hunk 1]
```diff
@@ -1,45 +1,145 @@
...
+function getBodyClasses()
+{
+    $body_classes = '';
+    $screen_sizes = ['xs', 'sm', 'md', 'lg', 'xl'];
+
+    // Add classes related to the "sidebar_mini" configuration.
+
+    if (config('adminlte.sidebar_mini', true) === true)
+        $body_classes .= 'sidebar-mini ';
+    elseif ($sidebar_mini_cfg == 'md')
```
Comment (REJack) [inline resources/views/page.blade.php:19]: Where comes `$sidebar_mini_cfg` from?

Comment (Shidersz) [reply resources/views/page.blade.php:19]: It comes from a previous implementation like this:
```
$sidebar_mini_cfg = config('adminlte.sidebar_mini', true);

if ($sidebar_mini_cfg === true)
    $body_classes .= 'sidebar-mini ';
elseif ($sidebar_mini_cfg == 'md')
```
Going to fix this... thanks!

**Comment by REJack** [comment]
> Thanks for your PR! Can we cleanup the file to move the php functions to an smarter solution like a blade view modifier?

@lacodimizer What you mean with view modifier?

**Comment by Shidersz** [comment]
@REJack I agree with @lacodimizer that all this PHP logic should be moved to other place, maybe a helper file for the blade layouts, but I'm not familiarized with the source code structure. Any suggestion on where to move all this logic?

**Comment by lacodimizer** [comment]
@Shidersz you can add the functions as methods in the class src/AdminLte.php. So you can call this methods with $adminlte->getBodyClasses() in the page.blade.php.

Similar is used for the menu.
https://github.com/jeroennoten/Laravel-AdminLTE/blob/master/resources/views/page.blade.php#L95

**Comment by lacodimizer** [comment]
@Shidersz @REJack i have pushed the changes, can you test it please?

**Comment by REJack** [comment]
I will test this tomorrow in the new AdminLTE Page CMS Project that I recently started to code 😄.

**Comment by Shidersz** [comment]
I tried these changes on a local project and got the next error:

> Facade\Ignition\Exceptions\ViewException
> Class 'JeroenNoten\LaravelAdminLte\View' not found (View: /var/www/html/laradmin/resources/views/vendor/adminlte/page.blade.php) 

That references to the **line 54** of the file **AdminLte.php**

```
if (config('adminlte.layout_topnav') || View::getSection('layout_topnav')) {
    $body_classes[] = 'layout-top-nav';
}
```

**Comment by Shidersz** [comment]
Also, had to change method **getBodyData()** like this: 

```
public function getBodyData()
{
    $body_data = [];

    // Add data related to the "sidebar_scrollbar_theme" configuration.

    $sb_theme_cfg = config('adminlte.sidebar_scrollbar_theme', 'os-theme-light');

    if ($sb_theme_cfg != 'os-theme-light') {
        $body_data[] = 'data-scrollbar-theme='.$sb_theme_cfg;
    }

    // Add data related to the "sidebar_scrollbar_auto_hide" configuration.

    $sb_auto_hide = config('adminlte.sidebar_scrollbar_auto_hide', 'l');

    if ($sb_auto_hide != 'l') {
        $body_data[] = 'data-scrollbar-auto-hide='.$sb_auto_hide;
    }

    return trim(implode(' ', $body_data));
}
```
Because the previous implementation was returning a wrong string.

**Comment by lacodimizer** [comment]
@Shidersz is fixed

**Comment by Shidersz** [comment]
@lacodimizer Nice, thanks, I did not found another issue. Just to know, how do you push your commits to this particular PR?

**Comment by REJack** [comment]
@Shidersz You can allow him to push in your Branch/PR. The default setting is allow maintainers to push in your branch.

**Comment by Shidersz** [comment]
Yea, I know that, I'm asking about the procedure, I'm new to Github. For what I'm reading I believe I can fetch the PR branch from this project (my upstream) and then push commits into that branch, is that right?

Previously I was pushing commits into the branch of the forked repository [Shidersz:improve_page_blade_structure](https://github.com/Shidersz/Laravel-AdminLTE/tree/improve_page_blade_structure), but now that branch is outdated and adding changes on it may generate conflicts.

I'm just learning how to do the things in the right way.

**Comment by lacodimizer** [comment]
@Shidersz it's already pushed to your branch, you can see that in the commits / file changes in this pr :)

**Comment by Shidersz** [comment]
@lacodimizer @REJack  Wow, sorry guys, I'm learning how to use Github. Anyway, I was ALMOST sure I have reviewed my branch and didn't see the commits :-)

**Pull request merged**